### ISSUE TITLE
"error: constructor Content in class Content" for release 2.2.21

### ISSUE THREAD

**Issue by jamescoggan** [aws/aws-sdk-android#179]
Version 2.2.21 has changed all these constructors that my app uses.
I suspect this shouldn't be done on a patch version? at least a minor release?
AWS Documentation recommends the usage of `2.+` in the `build.gradle` file but due to this change I had to force my `build.gradle` to `2.2.21` to avoid breaking my builds.

All the methods bellow now give this error:

```
Error:(59, 34) error: constructor Content in class Content cannot be applied to given types;
required: no arguments
found: String
reason: actual and formal argument lists differ in length
```

My code: All lines are invalid with `2.2.21`
```
Content subjectContent = new Content(title);
Body messageBody = new Body(new Content(body));
Message feedbackMessage = new Message(subjectContent, messageBody);
Destination destination = new Destination().withToAddresses(emailTo);
SendEmailRequest request = new SendEmailRequest(emailFrom, destination, feedbackMessage);
```

**Comment by fosterzhang** [comment]
@jamescoggan Yes this shouldn't happen. Thanks for reporting. We are looking into this and get it fixed ASAP.

**Comment by karthiksaligrama** [comment]
@jamescoggan this is fixed with 2.2.22
